### PR TITLE
Improved upgrade page [MAILPOET-5142]

### PR DESCRIPTION
**Pull request by NeosinneR**
## Description

Updated the upgrade page (`admin.php?page=mailpoet-upgrade`):

- Removed  `Fuel growth with a MailPoet Business plan` section.
- Removed the Agency plan section.
- Made the background white.
- Moved the top section title and description into the Business/Creator plan section.
- Updated plan features to correspond with [the pricing page](https://account.mailpoet.com/).
- Added top bar with MailPoet logo (no Updates button though). 

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

MAILPOET-5142

## After-merge notes

_N/A_

## Tasks

- [x] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [x] 🚫 I added sufficient test coverage
- [x] 🚫 I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

### CODE REVIEW

File: mailpoet/views/upgrade.html [hunk 1]
```diff
@@ -1,53 +1,33 @@
...
-          <%= _x('Upgrade', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %>
-        </a>
-      </div>
-      <div class="mailpoet-gap-large"></div>
+<style>
+  body { background: white; }
+  #wpbody-content { padding-top: 64px; }
+</style>
+<div class="mailpoet-top-bar">
+  <a role="button" class="mailpoet-top-bar-logo" title="Back to section root" tabindex="0" href="admin.php?page=mailpoet-homepage">
```
Comment (websupporter) [inline mailpoet/views/upgrade.html]: `title` attribute should be translateable

**Review by websupporter — CHANGES_REQUESTED** [review]
thank you @neosinner. lgtm. just one small fix.

File: mailpoet/views/upgrade.html (hunk 1 above)
Comment (NeosinneR) [reply mailpoet/views/upgrade.html]: Good catch, fixed

**Review by websupporter — APPROVED** [review]
lgtm. thank you @NeosinneR

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**